lib/index.js

```diff
@@ -101,81 +41,248 @@ const scrollOptions = {
      this.rotate = config.rotate
      this.flip = config.flip
      this.font = getFont(config.font)
+     this.transform = config.transform
+     this.reverseDisplayOrdering = config.reverseDisplayOrdering
+   }
+   
+   setController (requested = 0) {
+     if (requested < 0 || requested >= this.controllerCount) { 
+       throw new Error(`Requested controller ${requested} is not configured.`)
+     }
+     this.activeController = requested
    }
 
    async displayTest (controller, enable) {
-     setController(this, controller)
-     await shiftOut(this, Registers.DisplayTest, enable ? on : off)
+     if (undefined === enable) {
```

Comment:
you can just use `if (!enable)` here. In terms of best practices, the best way to keep this block clean is to have the positive condition first, and drop the else:
```js
if (enable) {
  this.setController(controller)
  await this.shiftOut(Registers.DisplayTest, enable ? on : off)
  return
}

enable = controller;
for (controller = 0; controller < this.controllerCount; controller++) {
  this.setController(controller)
  await this.shiftOut(Registers.DisplayTest, enable ? on : off)
}
```

Reply:
Functions were created.  Please review latest comments.